DIA/UdtKind.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace DIA
+{
+    public enum UdtKind
+    {
+        UdtStruct,
```

Comment:
Maybe Udt is not needed at the beginning of the enumeration value names as it is already in the enumeration name (UdtKind). For example: change UdtStruct to Struct...

Reply:
Same comment as before, I try to respect the current coding style of the library, but I do prefer clean C# name; so let's fix it :)